packages/chat-component/src/styles/chat-component.ts

```diff
@@ -251,14 +292,14 @@ export const chatStyle = css`
     justify-content: end;
   }
   .tab-component__content {
-    padding: 20px 20px 20px 0px;
+    padding: var(--d-base) var(--d-base) var(--d-base) 0;
   }
   .tab-component__paragraph {
     font-family: monospace;
-    font-size: large;
-    border: 1px solid var(--light-gray);
-    border-radius: 25px;
-    padding: 20px;
+    font-size: var(--font-large);
+    border: var(--border-thin) solid var(--c-light-gray);
+    border-radius: var(--radius-large);
+    padding: var(--d-base);
   }
   .chat-history__footer {
```

Comment:
looks like it's still here, but something is causing an issue for it to render...

Reply:
I could not see the issue, maybe it has updated?